lib/ansible/modules/windows/win_uri.py

```diff
@@ -178,6 +178,11 @@
 '''
 
 RETURN = r'''
+elapsed:
+  description: The number of seconds that elapsed while performing the download
+  returned: always
+  type: int
```

Comment:
This is actually a float and can return decimal values.

Reply:
Then we have it mispresented for win_reboot too :-)
Will fix !